terraform/bambictf.tf

```diff
@@ -48,6 +50,12 @@ data "hcloud_image" "bambiengine" {
   most_recent   = true
 }
 
+data "hcloud_image" "bambimoloch" {
+  with_selector = local.engine_count > 0 ? "type=bambimoloch" : null

```

Comment:
This needs to be `elk_count` and `type=bambielk` (line below as well)

Reply:
changed